lib/models/note.js

```diff
@@ -223,6 +228,18 @@ module.exports = function (sequelize, DataTypes) {
 
   Note.parseNoteId = function (noteId, callback) {
     async.series({
+      parseNoteIdByArchivedAlias: function (_callback) {
```

Comment:
I'm not totally sure about why we keep the previously used alias in track?
What if the archived alias you about to change has been used?

Reply:
> I'm not totally sure about why we keep the previously used alias in track?
My thought is to avoid some user saved the archived alias but can't connect the note. 

> What if the archived alias you about to change has been used?
The user just change one that not be used. Or we need to set the special and personal prefix for the alias?

Reply:
To make this PR specific, it's better not to add this feature for redirecting previous alias to new one.
I suggest to remove this and just adding a notice alert area in the custom note url modal like: `Notice: the previous custom note url will not be redirected to the new one, please take your own risk to change this.`